x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/settings/default_settings_flyout.tsx

```diff
@@ -56,6 +56,7 @@ export const DefaultSettingsFlyout: React.FC<DefaultSettingsFlyoutProps> = ({ cl
     reduce_whitespace: reduceWhitespace,
     run_ml_inference: runMLInference,
   } = pipelineState;
+  const elementRef = useRef<HTMLAnchorElement>(null);
```

Comment:
can we use better name here, e.g. `firstFocusInFlyoutRef` so that folks know what's going on? Perhaps worht adding a short comment that it's purely for a11y reasons here

Reply:
Done using:
```typescript
  // Reference the first focusable element in the flyout for accessibility on click or Enter key action either Reset or Save button
  const firstFocusInFlyoutRef = useRef<HTMLAnchorElement>(null);
```